.github/workflows/centraldb_angular_frontend_check.yaml

```diff
@@ -0,0 +1,34 @@
+name: CentralDashboard-angular Frontend check
```

Comment:
Let's rename this to `CentralDashboard-angular Frontend Tests` and also the file to `centraldb_angular_frontend_test.yaml`